build.gradle

```diff
@@ -998,6 +998,7 @@ project(':clients') {
     testCompile libs.bcpkix
     testCompile libs.junit
     testCompile libs.mockitoCore
+    testCompile libs.log4j
```

Comment:
Why is this needed? Supposedly the `slf4jlog4j` dependency brings this too?

Reply:
I guess you're doing it because we refer to log4j at compile time now?

Reply:
yes, it's an import for `LogCaptureAppender`